pkg/qemu/qemu.go

```diff
@@ -31,6 +31,7 @@ type Config struct {
 	Name         string
 	InstanceDir  string
 	LimaYAML     *limayaml.LimaYAML
+	Driver       *LimaQemuDriver
```

Comment:
With this change the whole `Config` struct becomes redundant because all other fields are just copies from the `Driver` struct.

Would it make sense to get rid of `Config` and just pass around `Driver` instead?

Reply:
I have not properly thought this through; it just occurred to me that there was no additional information in `Config` beyond what is already in `Driver`. In general I would try to avoid keeping the same data in multiple places because then it can get out of sync, and hard to debug...

Maybe wait for feedback from @AkihiroSuda before making this change?

Reply:
If this `Driver` addition is just for getting the driver name, can we just get the driver name from the yaml?

Reply:
We could, but what is the reason for having a separate `Config` struct? The base `Driver` is just pointers to instance data and YAML:

```go
type BaseDriver struct {
	Instance *store.Instance
	Yaml     *limayaml.LimaYAML
}
```